src/colors.scss

```diff
@@ -1,12 +1,12 @@
+/* purgecss start ignore */
```

Comment:
[sanity check] purgecss won't keep these CSS variables around without these annotations to ignore, even though the CSS variables are now explicitly referenced in JavaScript?